index.d.ts

```diff
@@ -729,6 +730,16 @@ declare namespace Dysnomia {
     selfStream: boolean;
     selfVideo: boolean;
   }
+  interface VoiceChannelEffect {
+    guild: Guild | Uncached;
+    channel: TextVoiceChannel | Uncached;
```

Comment:
makes it consistent with the JSDoc
```suggestion
    channel: VoiceChannel | Uncached;
```

Reply:
Types wise, this should be fine? The reason I use `TextVoiceChannel` is to allow for text methods to be used, this already extends `VoiceChannel` and `AnyVoiceChannel` would suggest to use this.
```ts
type AnyVoiceChannel = TextVoiceChannel | StageChannel;
```

Reply:
oh! let's use `AnyVoiceChannel` then (even though stage channels don't have them, I think it might be more suitable)